DigitalOffice/Services/FileService/Models/File.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace LT.DigitalOffice.FileService.Models
+{
+    public class File
```

Comment:
What is it?

Reply:
Dto, as found in UserService from develop.

Reply:
Hm... I thought we adding a _Response_ ending